corehq/apps/api/resources/messaging_event/utils.py

```diff
@@ -39,7 +39,7 @@ def get_order_by_field(request_params):
     order_by = request_params.get("order_by")
 
     if not order_by:
-        return "date"
+        return "date_last_activity"
 
     if order_by not in ("date", "-date", "date_last_activity", "-date_last_activity"):
```

Comment:
Will we need two indexes, one for `date` and another for `date_last_activity`?

Reply:
Ideally yes, but since the DET only uses `date_last_activity` I'm only adding that one now. I'm on the fence about removing option for ordering by `date` entirely.

Reply:
I think we should either add the extra index or remove the option to filter by `date`.

Reply:
I have removed the option to filter by `date`